src/components/tools/Partials/Dashboard/View.svelte

```diff
@@ -0,0 +1,53 @@
+<script>
+  export let useTabs = false;
+</script>
+
+<!-- Chart slots -->
+{#if $$slots.chart_title}
```

Comment:
TIL that `$$slots` is a special variable that is an object with the name of slots being passed to a component ([link to docs](https://svelte.dev/docs#template-syntax-slot-$$slots)). It's useful for only rendering a slot or other content if the parent component uses the slot.